src/test/data/JsonAddressBookStorageTest/invalidPersonAddressBook.json

```diff
@@ -4,5 +4,7 @@
     "phone": "9482424",
     "email": "hans@example.com",
     "address": "4th street"
-  } ]
+  } ],
+  "elderly": [],
+  "volunteers": []
```

Comment:
Are we not putting these in a separate json file?

Reply:
Yup my bad